lib/node_modules/@stdlib/utils/error-to-json/README.md

```diff
@@ -0,0 +1,241 @@
+# Error to JSON
+
+> Return a [JSON][json] representation of an [error][js-error] object.
+
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var toJSON = require( '@stdlib/utils/error-to-json' );
+```
+
+#### toJSON( TODO )
+
+Returns a [JSON][json] representation of an [`error`][js-error] object.
+
+``` javascript
+var err = new Error( 'beep' );
+
+var json = toJSON( err );
+/* returns
+    {
+        "type": "Error",
+        "name": "Error", // if present
+        "message": "beep",
+        "stack": "<stack>" // if present
+    }
+*/
+```
+
+The [JSON][json] `object` is __guaranteed__ to have the following properties:
+
+* __type__: error type
+* __message__: error message
+
+The __only__ standardized cross-platform property is `message`. Depending on the platform, the following properties __may__ be present:
+
+* __name__: error name
```

Comment:
Can we end each property description with a period? This is our common practice for `options`.